package.json

```diff
@@ -48,12 +48,12 @@
         "@testing-library/user-event": "^14.2.5",
         "@wojtekmaj/enzyme-adapter-react-17": "^0.6.1",
         "autoprefixer": "^10.4.7",
-        "babel-jest": "^26.3.0",
-        "babel-loader": "^8.0.6",
+        "babel-jest": "^29.3.0",
```

Comment:
i dont know if it makes a difference, but i think i would keep this on the same version as `jest`? unfortunately, we are stuck on `jest@26` for now